lib/rules/aria-empty-table-header.json

```
@@ -0,0 +1,13 @@
+{
+	"id": "aria-empty-table-header",
+	"selector": "th, [role=\"rowheader\"], [role=\"columnheader\"]",
+	"matches": "heading-matches",
+	"tags": ["wcag131"],
+	"metadata": {
```

The rule should mark empty header elements for needs review but the check produces failure results. Adding `reviewOnFail: true` as a property of the rule should fix it.

```suggestion
  "reviewOnFail": true,
	"metadata": {
```